### DISCUSSION TITLE
Request for Inclusion of IOC/FioCruz as Academic Institutions for Access to ROSIE Webserver

### DISCUSSION THREAD

**Discussion by gomes-netof** [RosettaCommons/rosetta#82] [ROSIE]
Dear Rosie team

I am writing on behalf of the Instituto Oswaldo Cruz (IOC/FioCruz-RJ) to kindly request that our institutional domains (@ioc.fiocruz.br and @fiocruz.br) be included as recognized academic institutions for access to the ROSIE web server.

The complete address of our institution is:
Instituto Oswaldo Cruz (IOC/FioCruz-RJ) – Campus Maré
https://www.ioc.fiocruz.br/
Centro de Pesquisa, Inovação e Vigilância em Covid-19 e Emergências Sanitárias (CPIVC19)
Bloco 2 – Sala 2-033
Ave. Brasil, 4036 - Manguinhos
CEP 21040-361
Rio de Janeiro, RJ, Brasil

As one of Brazil's leading research institutes, particularly in public health and biomedical research, access to the ROSIE webserver would greatly benefit our ongoing research and academic activities. Our teams are dedicated to advancing scientific knowledge, particularly in the context of COVID-19, snake bite envenomation, and other emergent health threats, and access to ROSIE's computational resources would be invaluable in our efforts.

Francisco Gomes-Neto
Laboratório de Toxinologia - LATOX - IOC - FioCruz

**Comment by lyskov** [comment]
Thank you for letting us know, - this should be fixed now! Please feel free to re-open this ticket if you still having difficulties.
